docs/csharp/whats-new/csharp-9.md

```diff
@@ -61,7 +61,17 @@ The compiler synthesizes different versions of the methods above. The method sig
 
-Records also have a synthesized constructor and a "clone" method for creating copies. The synthesized constructor has one argument of the record type. It produces a new record with the same values for all properties of the record. This constructor is private if the record is sealed, otherwise it's protected. The synthesized "clone" method supports copy construction for record hierarchies. The term "clone" is in quotes because the actual name is compiler generated. You can't create a method named `Clone` in a record type. The synthesized "clone" method returns the type of record being copied using virtual dispatch. If a record type is `abstract`, the clone method is also `abstract`. If a record type is `sealed` and its base type isn't `object`, the clone method is `sealed`. If the base type of the record is `object`, the clone method is `virtual` if the record type isn't `sealed`. Otherwise, it's `override`. The result of all these rules is the equality is implemented consistently across any hierarchy of record types. Two records are equal to each other if their properties are equal and their types are the same, as shown in the following example:
+Records also have a synthesized constructor and a "clone" method for creating copies. The synthesized constructor has one argument of the record type. It produces a new record with the same values for all properties of the record. This constructor is private if the record is sealed, otherwise it's protected. The synthesized "clone" method supports copy construction for record hierarchies. The term "clone" is in quotes because the actual name is compiler generated. You can't create a method named `Clone` in a record type. The synthesized "clone" method returns the type of record being copied using virtual dispatch. The compiler adds different modifiers for the "clone" method depending on the access modifiers on the `record`:
+
+- If the record type is `abstract`, the "clone" method is also `abstract`.
+- For record types that aren't `abstract` when the base type is `object`:
+  - If the record is `sealed`, no additional modifiers are added to the "clone" method (meaning it is not `virtual`).
+  - If the record isn't `sealed`, the "clone" method is `virtual`.
+- For record types that aren't `abstract` when the base type is not `object`:
+  - If the record is `sealed`, the "clone" method is also `sealed`.
+  - If the record isn't `sealed`, the "clone" method is `override`.
```

Comment:
A final bit, if it's abstract and the base type is not object, clone method is both "abstract" and "override"

https://sharplab.io/#v2:EYLgtghgzgLgpgJwD4AEDMACFAmDBhAWACgBvYgX2OPQwmFgQgGMYME4mB7BAEwwEFiZIpSLVMdBs1bsuvDACEMIAUIpA===

```
    [System.Runtime.CompilerServices.NullableContext(1)]
    public abstract override A <Clone>$();

    protected B([System.Runtime.CompilerServices.Nullable(1)] B original)
        : base(original)
    {
    }
```